packages/help-center/src/components/utils.tsx

```diff
@@ -14,3 +15,45 @@ export const generateContactOnClickEvent = (
 		} );
 	}
 };
+
+export const getFilteredConversations = ( {
+	conversations,
+}: {
+	conversations: ZendeskConversation[];
+} ) => {
+	const recentConversations: ZendeskConversation[] = [];
+	const archivedConversations: ZendeskConversation[] = [];
+
+	if ( ! conversations ) {
+		return {
+			recentConversations,
+			archivedConversations,
+		};
+	}
+
+	if ( Array.isArray( conversations ) ) {
+		conversations.forEach( ( conversation: ZendeskConversation ) => {
+			if ( ! conversation?.metadata?.createdAt ) {
+				recentConversations.push( conversation );
+				return;
+			}
+
+			const createdAt = conversation.metadata?.createdAt;
+			const createdAtDate = new Date( createdAt as string | number | Date );
+			const now = new Date();
+			const oneYearAgo = new Date( now.setFullYear( now.getFullYear() - 1 ) );
+
+			if ( createdAtDate < oneYearAgo ) {
+				archivedConversations.push( conversation );
+			} else {
+				recentConversations.push( conversation );
+			}
+			return;
```

Comment:
Can you test if it is necessary to have this return?

Reply:
This is not needed as well. I removed the return statement.